Report.md

```diff
@@ -698,10 +704,37 @@ The group has two scripts to automate job setup and runs:
     - Avg time/rank
     - Total time
     - Variance time/rank
-
 All of these metrics were measured and recorded by Caliper. They are available
 in each run's respective `.cali` file.
 
+#### Radix Sort Performance Evaluation:
```

Comment:
Now that this header doesn't have any text directly below it, maybe the other Radix headers should be demoted a step to indicate they're part of this section?

Reply:
Yes, when I was moving stuff around, I thought of that, but it slipped my mind. I'll get that done!